airflow/utils/log/filename_formatter.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+
+def format(task_instance, **kwargs):
```

Comment:
This seems overkill to me. You could just allow the use of dag_id, try_number, execution_date etc in a string so you could specify in the dict "/var/log/airflow/tasks/{dag_id}/{task_id}.{execution_date}.{try_number}.log". Or anything you like